opencti-platform/opencti-front/src/private/components/data/connectors/Connector.jsx

```diff
@@ -382,7 +414,167 @@ const ConnectorComponent = ({ connector, relay }) => {
                   )}
                 </Grid>
               )}
+              <Security needs={[SETTINGS_SETACCESSES]}>
+                <Grid item xs={6}>
+                  <Typography variant="h3" gutterBottom={true}>
+                    {t_i18n('Associated user')}
+                  </Typography>
+                  {connector.connector_user ? (
+                    <FieldOrEmpty source={connector.connector_user}>
+                      <List>
+                        {[connector.connector_user].map((user) => (userHasSettingsCapability ? (
```

Comment:
since `connector_user` is protected anyway with this capability on the API side, why do we add this test here in the frontend ? connector.connector_user would be null if the capability is not present, right ? 
Also, why put connector.connector_user in a list ?